acquia_lift.admin.inc

```diff
@@ -287,36 +304,101 @@ function acquia_lift_admin_form_submit($form, &$form_state) {
  * @param stdClass $option_set
  *   (optional) The content variation to show in the default view.
  */
-function acquia_lift_report($form, &$form_state, $agent_data, $option_set = NULL, $nested = FALSE) {
-  $legacy = FALSE;
-  if (acquia_lift_is_targeting_agent($agent_data)) {
-    // Display a report for any nested test.
-    $report = array();
-    $option_set = acquia_lift_get_option_set_for_targeting($agent_data->machine_name);
-    if (empty($option_set->targeting)) {
-      return $report;
+function acquia_lift_report($form, &$form_state, $agent_data) {
+  $form = array(
+    '#prefix' => '<div id="acquia-lift-reports">',
+    '#suffix' => '</div>',
+    '#attached' => array(
+      'css' => array(
+        drupal_get_path('module', 'acquia_lift') . '/css/acquia_lift.admin.css',
+      ),
+      'js' => array(
+        drupal_get_path('module', 'acquia_lift') . '/js/acquia_lift.admin.js',
+      ),
+      'library' => array(
+        array('acquia_lift', 'acquia_lift.reports'),
```

Comment:
Seems like we should just add admin.css and admin.js to the reports library dependencies assuming its only used here anyway.